Buildings/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mo

```diff
@@ -0,0 +1,303 @@
+  connect(relHumOut_eps.port_b, senMasFraOut.port_a)
+    annotation (Line(points={{-100,0},{-110,0}}, color={0,127,255}));
+  connect(TDryBulOut1.port_b, relHumOut_dis.port_a)
+    annotation (Line(points={{-70,60},{-80,60}}, color={0,127,255}));
+  connect(relHumOut_dis.port_b, senMasFraOut1.port_a)
+    annotation (Line(points={{-100,60},{-110,60}}, color={0,127,255}));
+  connect(hexWetNTU_TX.port_b1, sinWat.ports[3]) annotation (Line(points={{-10,-28},
+          {20,-28},{20,17.3333},{30,17.3333}}, color={0,127,255}));
+  connect(souAir1.ports[1], hexWetNTU_TX.port_a2) annotation (Line(points={{120,
+          -100},{0,-100},{0,-40},{-10,-40}}, color={0,127,255}));
+  connect(sinAir.ports[3], hexWetNTU_TX.port_b2) annotation (Line(points={{-160,
+          -64.6667},{-150,-64.6667},{-150,-64},{-138,-64},{-138,-40},{-30,-40}},
+                                                           color={0,127,255}));
+  connect(souWat2.ports[1], hexWetNTU_TX.port_a1) annotation (Line(points={{-160,
+          -20},{-40,-20},{-40,-28},{-30,-28}}, color={0,127,255}));
+  connect(w_a2.y[1], conversion.XiDry) annotation (Line(points={{175,-24},{172,
+          -24},{172,-50},{196,-50},{196,-80},{191,-80}}, color={0,0,127}));
+  connect(conversion.XiTotalAir, souAir.Xi_in[1]) annotation (Line(points={{169,
+          -80},{158,-80},{158,-64},{142,-64}}, color={0,0,127}));
+  connect(conversion.XiTotalAir, souAir1.Xi_in[1]) annotation (Line(points={{
+          169,-80},{154,-80},{154,-104},{142,-104}}, color={0,0,127}));
+  connect(conversion.XiTotalAir, souAir2.Xi_in[1]) annotation (Line(points={{
+          169,-80},{158,-80},{158,36},{142,36}}, color={0,0,127}));
+  annotation (Diagram(coordinateSystem(preserveAspectRatio=true,
+    extent={{-200,-120},{200,120}})),
+    experiment(
+      StopTime=100,
+      Tolerance=1e-06),
+    __Dymola_Commands(
+    file="Resources/Scripts/Dymola/Fluid/HeatExchangers/Validation/WetCoilEffectivenessNTU.mos"
+  "Simulate and plot"),
+  Documentation(info="<html>
+<p>
+This model duplicates an example from Mitchell and Braun 2012, example SM-2-1
+(Mitchell and Braun 2012) to validate a single case for the
+<a href=\"modelica://Buildings.Fluid.HeatExchangers.WetCoilEffectivenessNTU\">
+Buildings.Fluid.HeatExchangers.WetCoilEffectivenessNTU</a>
+model.
+</p>
+<h4>Validation</h4>
+<p>
+The example is a steady-state analysis of a wet coil with constant air
+and water inlet temperature and mass flow rate, and an increasing air inlet
+humidity which triggers the transition from a fully-dry to a fully-wet regime.
+The reference used for validation is the published experimental data.
+A discretized wet coil model is also simulated for comparison.
+</p>
+<p>
+Note that the outlet air relative humidity may slightly exceed 100% when using
+the epsilon-NTU model.
+</p>
+<p>
+The slight deviations we find are believed due to differences in the tolerance
+of the solver algorithms employed as well as differences in media property
```

Comment:
If it is indeed because of the solver tolerance, then why not simply increasing the tolerance to 1E-8?

Reply:
The comment was from the process of using Michael O'keef's model and is obsolete. I simply deleted this comment.